File: c/phonebook.c
// Lecture 03 - "Algorithms"
// program 27
// Phonebook implementations

#include "cs50.c"
#include <stdio.h>
#include <string.h>

int main(int argc, string argv[])
{
    string names[] = {"Carter", "David", "John"};
    string numbers[] = {"+1-617-495-1000", "+1-617-495-1000", "+1-949-468-2750"};

    // Name to search for...
    string name = argv[1];

    // Search...
    for (int i = 0; i < 3; i++)
    {
        if (strcmp(names[i], name) == 0)
        {
            // Name matches...
            printf("Found! %s\n", numbers[i]);
            return 0;
        }
    }

    printf("Not found!\n");
    return 1;

    return 0;
}

File: python/phonebook.py
# Lecture 06 - "Python"
# Program 11

people = {
    "carter": "+1-617-495-1000",
    "david": "+1-617-495-1000",
    "john": "+1-949-468-2750",
}

name = input("name: ")

if name in people:
    print(f"Name: {name}, Number: {people[name]}")
else:
    print("Not found")
